src/paths/all_shortest_paths.c

```diff
@@ -127,6 +131,9 @@ int igraph_get_all_shortest_paths(const igraph_t *graph,
     /* paths will store the shortest paths during the search */
     IGRAPH_CHECK(igraph_vector_ptr_init(&paths, 0));
     IGRAPH_FINALLY(igraph_i_gasp_paths_destroy, &paths);
+    /* paths will store the shortest paths during the search */
```

Comment:
Please fix the comment: `"path_edge" will store the...`. Right now the comment is copied straight from three lines above and it does not reflect what `path_edge` will be used for.